package.json

```diff
@@ -1,38 +1,38 @@
 {
-  "name": "photoshop.d.ts",
-  "version": "0.4.0",
-  "description": "Typescript declarations for photoshop",
-  "main": "app.js",
-  "scripts": {
-    "build": "npm run compile ./app.ts",
-    "test:cc": "npm run compile ./dist/cc/**",
-    "test:cs6": "npm run compile ./dist/cs6/**",
-    "test": "npm run test:cc && npm run test:cs6",
-    "pretest": "npm run build",
-    "compile": "tsc --target ES5 --module commonjs",
-    "start": "node app.js",
-    "convert": "npm start cc && npm start cs6"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/felixschl/photoshop.d.ts"
-  },
-  "keywords": [
-    "photoshop",
-    "typescript",
-    "javascript",
-    "api"
-  ],
-  "author": "Felix Schlitter",
-  "license": "BSD-2-Clause",
-  "bugs": {
-    "url": "https://github.com/felixschl/photoshop.d.ts/issues"
-  },
-  "devDependencies": {
-    "bennu": "~17.2.2",
-    "fs-extra": "~0.13.0",
-    "lodash": "~2.4.1",
-    "typescript": "^1.8.9",
-    "wordwrap": "0.0.2"
-  }
-}
+    "name": "photoshop.d.ts",
+    "version": "0.4.0",
+    "description": "Typescript declarations for photoshop",
+    "main": "app.js",
+    "scripts": {
+        "build": "npm run compile ./app.ts",
+        "test:cc": "npm run compile ./dist/cc/**",
+        "test:cs6": "npm run compile ./dist/cs6/**",
+        "test": "npm run test:cc && npm run test:cs6",
+        "pretest": "npm run build",
+        "compile": "tsc --target ES5 --module commonjs",
+        "start": "node app.js",
+        "convert": "npm start cc && npm start cs6"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/felixschl/photoshop.d.ts"
+    },
+    "keywords": [
+        "photoshop",
+        "typescript",
+        "javascript",
+        "api"
+    ],
+    "author": "Felix Schlitter",
+    "license": "BSD-2-Clause",
+    "bugs": {
+        "url": "https://github.com/felixschl/photoshop.d.ts/issues"
+    },
+    "devDependencies": {
+        "bennu": "~17.2.2",
+        "fs-extra": "~0.13.0",
+        "lodash": "~2.4.1",
+        "typescript": "^2.4.2",
+        "wordwrap": "0.0.2"
+    }
+}
```

Comment:
Thanks for working on this, but this is a lot white-space change and hides the significant change. Also the file no longer ends with a newline (as indicated by the crossed out circle at bottom of page here on gh). I have no objections to using 4 spaces if that makes it easier on your end to deal with, but could you make two separate commits for that, so that significant changes do not get lost in noise?

Reply:
OK thanks for the note! Forgot about autoformat on json files - but it's fixed now with the latest commit!